src/app/common/DocumentationLink.vue

```diff
@@ -1,28 +1,29 @@
 <template>
-  <KButton
+  <a
     class="docs-link"
-    appearance="outline"
+    :href="props.href"
     target="_blank"
-    :to="props.href"
   >
-    <KIcon
-      icon="externalLink"
-      color="currentColor"
-      size="16"
-      hide-title
-    />
+    <img src="@/assets/images/book.svg?url">
```

Comment:
Shouldn't that be: https://kongponents.konghq.com/components/icon.html#icon-1 ?

Reply:
I’ve updated this now.